rmc/utils/simultaneous_breaks.py

```diff
@@ -458,7 +458,7 @@ def process_section_group(
         Default is False.
```

Comment:
On line 398-407, we should change to:
```
    group_ht_path = (
        f"{TEMP_PATH_WITH_FAST_DEL}/freeze{freeze}_dataproc_temp_chisq_group{count}.ht"
    )
    if save_chisq_ht:
        group_ht_path = f"{SIMUL_BREAK_TEMP_PATH}/freeze{freeze}/dataproc_temp_chisq_group{count}.ht"
    ht = ht.checkpoint(all_loci_chisq_ht_path, overwrite=True)
```
Similarly for `run_batches.py` lines 373-382